### PR TITLE
Fixed failing tests in forms-dev

### PR DESCRIPTION
**Pull request by raduiordache**
Fixes #<issue>

## Checklist:
- [x] My code follows the pykechain style
    - [x] I added type hinting to all functions
    - [x] I added documentation for all public functions
    - [x] I locally used `tox` to check for dists and docs errors
    - [x] My code passes the `pep8` and `flake8` linting checks and no warnings
    - [x] I removed unused imports, using `Code > Optimize Imports` in PyCharm
    - [x] I used [`black`](https://github.com/psf/black) to format the new code
- [x] I have added tests that prove my fix is effective or that my feature works
    - [x] I committed fresh test cassettes
    - [x] I have proper test coverage (don't decline the coverage of the test)
- [x] I asked another teammate to review the code
- [x] I update the Changelog.md with the appropriate changes.

### CODE REVIEW

File: tests/models/test_property_reference.py [hunk 1]
```diff
@@ -7,7 +7,7 @@
     PropertyType,
     FilterType,
     Multiplicity,
-    ActivityRootNames
+    ActivityRootNames, WorkflowCategory
```
Comment (jberends) [inline tests/models/test_property_reference.py:10]: Use black here

File: tests/test_widgets.py [hunk 2]
```diff
@@ -588,7 +588,7 @@ def test_add_card_widget_ke_chain_pages(self):
 
         self.assertEqual(
             len(self.wm),
-            9,
+            13,
```
Comment (jberends) [inline tests/test_widgets.py:591]: Can we alter tge test to avoid len?

**Review by jberends — CHANGES_REQUESTED** [review]

File: tests/test_widgets.py (hunk 2 above)
Comment (raduiordache) [reply tests/test_widgets.py:591]: No,  this is Jelle's legacy and I am not touching it

File: tests/models/test_property_reference.py (hunk 1 above)
Comment (raduiordache) [reply tests/models/test_property_reference.py:10]: i have used african-american here

**Pull request merged**